src/main/resources/admin/widgets/LinkChecker/LinkChecker.html

```diff
@@ -6,32 +6,53 @@
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
       <style data-th-if="${css}" data-th-text="${css}"></style>
       <script src="https://unpkg.com/xlsx/dist/xlsx.full.min.js"></script>
-      <div class="link-checker container">
-        <div class="text-linkchecker">
-          <h2>LinkChecker</h2>
+      <div class="link-checker">
+        <div class="link-checker__info">
           <p>Check this content and its sub-content for broken links, both internal and external</p>
         </div>
-        <form id="form-link-checker" data-th-if="${serviceUrl}" method="post" data-th-action="${serviceUrl}">
+        <form id="link-checker__form" data-th-if="${serviceUrl}" method="post" data-th-action="${serviceUrl}">
             <input type="hidden" data-th-if="${key}" data-th-value="${key}" id="contentId"/>
         </form>
-        <button id="btn-link" onclick="startCheck()" value="Check">Check</button>
+        <div class="link-checker__start">
+          <button id="btn-start" onclick="startCheck()" value="Start">Start</button>
+          <div class="selection-radios">
+            <fieldset>
+              <legend>Check</legend>
+              <div>
+                <input type="radio" id="content"
+                name="selection" value="content" />
+                <label for="content">This content</label>
+              </div>
+              <div>
+                <input type="radio" id="children"
+                name="selection" value="children" />
+                <label for="children">Sub-content</label>
+              </div>
+              <div>
+                <input type="radio" id="both"
+                name="selection" value="both" checked/>
+                <label for="both">Both</label>
+              </div>
+            </fieldset>
+          </div>
+        </div>
         <button id="btn-stop" style="display: none;" onclick="window.stopCheck()" value="STOP">STOP</button>
         <button id="btn-download" onclick="window.downloadCSV()" style="display: none;" value="CSV"> <i class="fas fa-download"></i> Download Excel</button>
-        <div class="progress" style="display: none; width: 100%; position: relative;">
-          <!-- <div class="progress-bar-websocket">&nbsp;</div> -->
-          <div class="progress-container" style="height:100px">
-            <div class="count">Loading...</div>
-            <div class="circle-background"></div>
-            <div class="wrapper">
-              <div class="circle left"></div>
-              <div class="circle right"></div>
+        <div class="link-checker__progress">
+          <div class="circle-container">
```

Comment:
Please use semantic naming for elements and functions.
"Circle" is a visual description. For instance, "ProgressIndicator"/"progress-indicator" is an example of semantic naming.

Generally, we want names to describe the the functionality of an element, not what it looks like.